_content/articles/japanese-priests-obon_haseo-daien-tsutomu.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "A Japanese Priest's Obon"
+authors:
+  - "Daien Tsutomu Haseo" 
+external_url: "https://bschawaii.org/shindharmanet/wp-content/uploads/sites/3/2012/04/Haseo-Priest.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1X-K37UM4CmiKdBEW7t37WQWRDdogYsge/view?usp=sharing"
+course: japanese
+tags:
+  - sangha
+  - ritual
+year: 2012
+journal: Shin Dharma Net
+publisher: Buddhist Study Center 
+address: "Hawaii"
+pages: 2
```

Comment:
Definitely essays for this one. I use the `reference` folder primarily for works that I don't expect people to read cover-to-cover (e.g. chanting books) or for things that are genuinely hard to classify (e.g. an interactive map).